<!-- TODO: Insert components of acting component-->
<launch>
    <arg name="role_name" default="ego_vehicle" />
    <arg name="control_loop_rate" default="0.1" />

    <node pkg="acting" type="stanley_controller.py" name="stanley_controller" output="screen">
        <param name="control_loop_rate" value="$(arg control_loop_rate)" />
        <param name="role_name" value="$(arg role_name)" />
    </node>
</launch>
